### DISCUSSION TITLE
Gauge Java  0.7.14 not yet available in mvnrepository.com

### DISCUSSION THREAD

**Discussion by cyberjaime45** [getgauge/gauge#1962] [General]
Hello,

Any update on this?

**Comment by zabil** [comment]
This might take a week or two. Unfortunately the upload scripts failed and the binaries have to be published manually.

**Reply by zabil** [reply]
I took a look at this and it's not straight forward to manually publish the released gauge-java file. I think it's best to wait for the next release (0.7.15) to trigger an automated build to bump up the version and deploy to maven.

**Reply by sriv** [reply]
The build scripts are fixed, gauge-java 0.7.15 is now available in maven-central: https://mvnrepository.com/artifact/com.thoughtworks.gauge/gauge-java/0.7.15
